### DISCUSSION TITLE
New page in the docs: API/Groovy tutorial

### DISCUSSION THREAD

**Discussion by macmarrum** [freeplane/freeplane#1314] [General]
Further to [previous discussions in the subject](https://github.com/freeplane/docs/discussions/76), I've created a tutorial covering the basics of Groovy and scripting in Freeplane.

https://docs.freeplane.org/scripting/api-groovy-tutorial.html

**Comment by abc16361** [comment]
Thank you!

**Comment by dpolivaev** [comment]
Beautiful work

**Comment by euu2021** [comment]
Nice didactic sequence 👏👏

**Comment by macmarrum** [comment]
New section added: [Conditional Styles with Script Filters](https://docs.freeplane.org/scripting/api-groovy-tutorial.html#conditional-styles-with-script-filters)
